packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -73,6 +74,10 @@ class InkResponse extends StatefulWidget {
   /// The radius of the ink splash.
   final double radius;
 
+  /// The clipping radius of the containing rect. Used for ink responses that are not contained in [Material]
```

Comment:
If an ink response is inside a `Material` element then it will use the corner radius of the material object (2.0 pt). If not, it will always be a rect with no radius. Maybe I will just remove that comment as its irrelevant.